app/services/invite_schools.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class InviteSchools
+  def run(school_ids)
+    logger = Rails.logger
+    logger.info "Emailing schools"
+
+    school_ids.each do |school_id|
+      school = School.find_by(id: school_id)
+
+      if school.nil?
+        logger.info "School not found: #{school_id} ... skipping"
+        next
+      end
+
+      token = reference
+      recipient = recipient_email(school)
+
+      school.nomination_emails.create!(
+        token: token,
+        sent_to: recipient,
+        sent_at: Time.zone.now,
+      )
+      send_nomination_email(recipient, token)
```

Comment:
We need to record the ID back from notify

Reply:
I'm passing that token as a reference to notify. When it's time to poll we'll be using that token to identify the email